src/Field/NominatimSearch/NominatimSearch.jsx

```diff
@@ -200,6 +210,10 @@ export class NominatimSearch extends React.Component {
         this.doSearch();
       }
     });
+
+    if (!inputValue && onClear) {
```

Comment:
Note: This will evaluate the input value `0` as truthy. Not sure if this is intended.

Reply:
Shouldn't be a problem as the inputValue is of type `string` or `undefiend`.